src/python/pants/backend/python/goals/tailor.py

```diff
@@ -161,13 +161,14 @@ def add_req_targets(files: Iterable[str], alias: str, target_name: str) -> None:
     if python_setup.tailor_pex_binary_targets:
         # Find binary targets.
 
-        # Get all files whose content indicates that they are entry points.
+        # Get all files whose content indicates that they are entry points or are __main__.py files.
         digest_contents = await Get(DigestContents, PathGlobs, all_py_files_globs)
+        all_main_py = await Get(Paths, PathGlobs, req.search_paths.path_globs("__main__.py"))
```

Comment:
These two `Get`s could've been joined in a `MultiGet`, no?

Reply:
We really need a flake8 plugin for this 🤔